tests/queries/0_stateless/01277_unixTimestamp64_compatibility.sql

```diff
@@ -0,0 +1,64 @@
+WITH
+	toDateTime64('2019-09-16 19:20:12.345678910', 3) AS dt64
+SELECT
+	dt64,
+	fromUnixTimestamp64Milli(toUnixTimestamp64Milli(dt64)),
+	fromUnixTimestamp64Micro(toUnixTimestamp64Micro(dt64)),
+	fromUnixTimestamp64Nano(toUnixTimestamp64Nano(dt64));
```

Comment:
Maybe it would be better to reduce the number of these function by passing "Milli" or "Micro" as a parameter like this:

```
toUnitTimestamp64(dt64, "ms")
```

Reply:
Well, not sure what good that should bring, but it would complicate users' life by forcing to memorize that argument type/values, etc, and not being able to autocomplete with `TAB` while in `clickhouse-client`.

Reply:
It seems the name `toUnixTimestamp64Milli()` isn't clear enough without reading the documentation. We could rename it to `toUnixTimestamp64InMilliseconds()`, but it's too long name.

Reply:
We already have the function `toUnixTimestamp()`, maybe the best way is to make new functions look like an extension to `toUnixTimestamp()`. I mean we could add something like

```
toUnixTimestamp(dt [, second|s|millisecond|ms|microseconds|us|nanoseconds|ns] [, timezone])
toUnixTimestamp64(dt [, second|s|millisecond|ms|microseconds|us|nanoseconds|ns] [, timezone])
```

However doing it that way means more work to do.

Reply:
> second|s|millisecond|ms|microsecond**s**|us|nanosecond**s**|ns

That is exactly the problem, it is hard to type and hard to remember: there are many versions of how you can type 'milliseconds' in this context: "ms", "milli", "millisecond", "milliseconds". And since we don't have builtin help on arguments in `clickhouse-client` it is a bit tedious to deduce proper thing to type in.

Also, `toUnixTimestamp` currently does not support the -milli, -micro, -nano, etc, and hence I assume there was no need to do that (and no need to do it now other than "consistency"), moreover, updating current implementation would significantly complicate argument deduction.

Reply:
ok